<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Source Sans Pro</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="ArtWithMachineLearningLabel">
        <property name="font">
         <font>
          <family>Source Sans Pro</family>
          <pointsize>36</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Art With Machine Learning</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="OriginalStyleLabel">
        <property name="font">
         <font>
          <family>Source Sans Pro</family>
          <pointsize>36</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Original Styles</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="minimumSize">
         <size>
          <width>426</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>426</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;center&gt;Your Next Three Results Will Be&lt;br&gt; Ready In 20 Seconds&lt;/cenmter&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="LaMuseLabel">
        <property name="minimumSize">
         <size>
          <width>426</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>426</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Source Sans Pro</family>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Picasso's
La Muse</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="RainPrincessLabel">
        <property name="minimumSize">
         <size>
          <width>426</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>426</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Source Sans Pro</family>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Leonid Afremov's
Rain Princess</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>426</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>426</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Edvard Munch's
The Scream</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGraphicsView" name="graphicsView_Scream_Stylized_16">
        <property name="minimumSize">
         <size>
          <width>426</width>
          <height>240</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>426</width>
          <height>240</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/Logo/Logos/index.png);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGraphicsView" name="graphicsView_Scream_Stylized_18">
        <property name="minimumSize">
         <size>
          <width>426</width>
          <height>240</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>426</width>
          <height>240</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">Border-image: url(:/Resized/Resized Pictures/La Muse - Resized.jpg)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGraphicsView" name="graphicsView_Scream_Stylized_15">
        <property name="minimumSize">
         <size>
          <width>384</width>
          <height>216</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>426</width>
          <height>240</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">Border-image: url(:/Resized/Resized Pictures/Rain Princess - Resized.jpg)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGraphicsView" name="graphicsView_Scream_Stylized_20">
        <property name="minimumSize">
         <size>
          <width>384</width>
          <height>216</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>426</width>
          <height>240</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/Resized/Resized Pictures/Scream - Resized.jpg);</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="YourStylizedResultsLabel">
      <property name="font">
       <font>
        <family>Source Sans Pro</family>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Your Stylized Results!</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QGraphicsView" name="graphicsView_Scream_Stylized_13">
        <property name="minimumSize">
         <size>
          <width>426</width>
          <height>240</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>426</width>
          <height>240</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/Camera/Camera Photo/Input Picture.jpg);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGraphicsView" name="graphicsView_Scream_Stylized_19">
        <property name="minimumSize">
         <size>
          <width>426</width>
          <height>240</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>426</width>
          <height>240</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/Stylized/Stylized Pictures/Stylized La Muse.jpg);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGraphicsView" name="graphicsView_Scream_Stylized_17">
        <property name="minimumSize">
         <size>
          <width>426</width>
          <height>240</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>426</width>
          <height>240</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/Stylized/Stylized Pictures/Stylized Rain Princess.jpg);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGraphicsView" name="graphicsView_Scream_Stylized_14">
        <property name="minimumSize">
         <size>
          <width>426</width>
          <height>240</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>426</width>
          <height>240</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/Stylized/Stylized Pictures/Stylized Scream.jpg);</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>10</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../Images/ResourceFile.qrc"/>
 </resources>
 <connections/>
</ui>
